pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/action/pip/AbstractPipAction.java

```diff
@@ -65,15 +65,15 @@
         this.packageExcludeFilter = packageExcludeFilter;
     }
 
-    public void execute(PackageInfo packageInfo, List<String> extraArgs) {
+    public void execute(PackageInfo packageInfo, List<String> extraArgs, boolean allowBuildingFromSdist) {
```

Comment:
See also the comment on the other PR about using Java's ability to define two method signatures using the same name. Then you don't need to change the default calls and test calls. You can add new test calls of course as we can proceed with incremental changes.